cos_alerter/daemon.py

```diff
@@ -43,6 +44,11 @@ def parse_args(args: List[str]) -> argparse.Namespace:
         choices=list(LEVELS),
         help='Logging level. Overrides config value "log_level"',
     )
+    parser.add_argument(
+        "--config",
+        required=False,
+        help='Path to config file. Defaults to /etc/cos-alerter.yaml'
+    )
```

Comment:
I would prefer to have the default defined here with the `default=` arg.

Reply:
I could do that, but you cannot pass that parameter to the **Config** constructor, due to the object being instantiated in the other module.

I can't just move its instantiation because it's also used in `alerter.py`. Adding a `default=` here would be confusing IMHO, because it wouldn't be a *true* default (since the class Config wouldn't see that when instantiating the object)

Reply:
Not really. The `Config` object does not work at all until `config.reload()` is called. In fact, I feel like putting the default in the `Config` class is likely to cause a bug down the road since we could in theory start reading the default file when a custom config file path has been specified. I think it is important that the config file is never set before we check the cli args.

Reply:
Oh okay, I thought `Config` was supposed to work since instantiation! Then yes, I'll put the default here and remove `__init__` :)